feature/25_HOME_VIEW

Symphonie234 opened the pull request:
- Created a new home view after the user logged in.
- Added new route, css and js files for home view.

Symphonie234 committed:
added view file for home page with its associated css and js, and also updated the route

ChronoDevs commented on app/Http/Controllers/Auth/LoginController.php:
```diff
@@ -114,4 +115,15 @@ protected function registerOrLoginUser($data)
         }
     }
+
+    public function logout(Request $request)
```
add comment above method

ChronoDevs commented on app/Http/Controllers/HomeController.php:
```diff
@@ -13,7 +13,7 @@ class HomeController extends Controller
      */
     public function __construct()
```
remove whole method __construct() if not use

Symphonie234 replied:
Understood!

ChronoDevs commented on routes/web.php:
```diff
@@ -2,6 +2,7 @@
 
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\Auth\LoginController;
+use App\Http\Controllers\Auth\LogoutController;
```
can't find LogoutController?

ChronoDevs commented on routes/web.php:
```diff
@@ -16,11 +17,10 @@
 */
 
+// Route localhost to home view
 Route::get('/', function () {
```
home route is duplicate.

Symphonie234 replied:
Understood!

Symphonie234 committed:
adjusted to the requested changes on the PR

Symphonie234 commented:
- Added comment in the **LoginController** on the logout method.
- Removed the unnecessary use of **LogoutController** namespace in the web.php.
- Removed the duplicate redirection to home in the web.php.
